### ISSUE TITLE
Multiple Home Kit Notifications on Arm

### ISSUE THREAD

**Issue by readylmay** [dustindclark/homebridge-envisalink#100]
Hey Dustin,

The issue on #90 is fixed in 1.0.14 - thanks you!

Quick side effect - I get 3 HomeKit notifications when arming for stay and bypassing a zone:

1.Alarm was armed for away.
2.Alarm was armed for stay.
3.Alarm was armed for Away.

I am assuming its from these three events:

```
[27/02/2022, 17:23:00] [Envisalink] Partition 1 is armed with one or more zones bypassed
[27/02/2022, 17:23:01] [Envisalink] Partition 11 is Armed
[27/02/2022, 17:23:02] [Envisalink] Partition 1 has been armed by Quick Arm, Auto Arm, Keyswitch, DLS or Wireless Key
```

**Comment by lfiocco** [comment]
I am getting the same thing.

alarm set to Home, Away, Stay.

wondering if this is a setting on the board that is not allowing night mode? 

Stay = arm, no IR, with delay
Night = arm, no IR no delay
Away = arm, include IR motion

**Comment by dustindclark** [comment]
Fixed in 1.1.4
